conf/timeseries-128shards-source.conf

```diff
@@ -1,20 +1,5 @@
     dataset = "prometheus"
-
-    definition {
-      partition-columns = ["tags:map"]
-      data-columns = ["timestamp:ts", "value:double:detectDrops=true"]
-      row-key-columns = [ "timestamp" ]
-      downsamplers = [ ]
-    }
-
-    options {
-      shardKeyColumns = [ "__name__", "_ns" ]
-      ignoreShardKeyColumnSuffixes = { "__name__" = ["_bucket", "_count", "_sum"] }
-      valueColumn = "value"
-      metricColumn = "__name__"
-      ignoreTagsOnPartitionKeyHash = [ "le" ]
-      copyTags = { }
-    }
+    schema = "prometheus"
```

Comment:
Can you please remind what this will evolve into when there are multiple schemas ?

Reply:
This line will be gone when we move to multi-schema, or will be a restricted set of schemas that each dataset is allowed to have.  I think you prefer restrictions (though that can be added later too)